addons/core/addon-test-support/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './dictionary';
+export * from './dom-helpers';
+export * from './helpers';
+export * from './labels';
+export * from './regex';
+export * from './types';
```

Comment:
Note that we are basically making all the exports here public. Mabe that's too mcuh, and we should trim that down to the very least user's will actually need?

Reply:
There's a user and there's an addon creator. The latter may need pretty much anything. So I don't think we have a choice.

Reply:
I have one concern with these: by looking at the `index.ts` file it is not clear which exports are available. You have to look into numerous private files to learn names of available modules.

So lets reexport everything explicitly, please!

Reply:
Hm, so when we want to be cautious with making APIs public, exporting them explicitly seems reasonable. But that this contradicts your previous statement. When exposing *everything*, the current seems easier to maintain and less error prone, as it's easy to forget about adding that explicit export. And tests might not cover that, e.g. when you accidentally import `foo` from `@ember-bdd/core/foo` instead of from index.

Also for TS users, itÄs easier to just let you IDE tell you what's available, instead of looking at the code?

I don't want to die on that hill, I am just saying there are tradeoffs... so your call!?

Reply:
That's why I initially prefered to stay with magic-less import paths. Each import goes from a single location, according to framework conventions, without reexports or Broccoli manipulations. Imports are a bit verbose but 100% predictable and 100% reliable.

***

> for TS users, itÄs easier to just let you IDE tell you what's available, instead of looking at the code?

Not everyone is using TypeScript, and forcing users to look in private folders for public imports feels wrong.